deploy/helm/tracee/templates/daemonset.yaml

```diff
@@ -30,14 +30,6 @@ spec:
         - name: tracee
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
-          {{- if .Values.everythingIsAnEvent }}
-          command: ["/tracee/tracee"]
-          args:
-            - --output
-            - json
```

Comment:
Don't we still need to set the output as json?

Reply:
This was all under the flag, which we are removing here. For the new binary, several things will change, and they are handled on the PR https://github.com/aquasecurity/tracee/pull/2786